README.md

```diff
@@ -1,16 +1,64 @@
+<img height='175' src="https://user-images.githubusercontent.com/37406965/51083189-d5dc3a80-173b-11e9-8ca0-28015e0893ac.png" align="left" hspace="1" vspace="1">
+
+# Self Service Android Application for MifosX
+
+An Android Application built on top of the MifosX Self-Service platform for end-user customers to view/transact on the accounts and loans they hold. Data visible to customers will be a sub-set of what staff can see. This is a native Android Application written in Java with POJO classes in Kotlin.
+
 ### Status
 
 [![Join the chat at https://gitter.im/openMF/self-service-app](https://badges.gitter.im/openMF/self-service-app.svg)](https://gitter.im/openMF/self-service-app?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://travis-ci.org/openMF/mifos-mobile.svg?branch=development)](https://travis-ci.org/openMF/mifos-mobile)
 
-![App Screenshots](https://user-images.githubusercontent.com/24931732/48102172-b0ccc800-e1de-11e8-9fb9-17c41234636e.png)
 
-# Self Service Android Application for MifosX
+## Screenshots
 
-An Android Application built on top of the MifosX Self-Service platform for end-user customers to view/transact on the accounts and loans they hold. Data visible to customers will be a sub-set of what staff can see. This is a native Android Application written in Java with POJO classes in Kotlin.
+<p>
+  <img src="https://user-images.githubusercontent.com/24931732/48102172-b0ccc800-e1de-11e8-9fb9-17c41234636e.png" />
+</p>
+
+## How to Contribute
+
+This is an OpenSource project and we would be happy to see new contributors. The issues should be raised via the GitHub issue tracker.
+For Issue tracker guidelines please click <a href="https://github.com/openMF/mifos-mobile/blob/development/.github/CONTRIBUTING.md">here</a>. All fixes should be proposed via pull requests.
```

Comment:
Fix the Issue template link: https://github.com/openMF/mifos-mobile/blob/development/.github/ISSUE_TEMPLATE.md